Sdext.tex

```diff
@@ -26,9 +26,9 @@ \section{Debug Mode} \label{debugmode}
     Full permission checks, or a relaxed set of permission checks, will apply
     according to \FdmAbstractcsRelaxedpriv.
 \item All interrupts (including NMI) are masked.
-\item Exceptions don't update any registers.  That includes {\tt cause}, {\tt
-    epc}, {\tt tval}, {\tt dpc}, and \Rmstatus. They do end execution of the
-    Program Buffer.
+\item When an exception occurs, pc may be updated, while all other registers must
```

Comment:
```suggestion
\item When an exception occurs, {\tt pc} may be updated, while all other registers must
```
We're probably not consistent on this in the spec, but the ISA spec uses that formatting.